src/main/java/com/google/sps/model/Trips.java

```diff
@@ -0,0 +1,10 @@
+package com.google.sps.model;
+
+import java.util.List;
+
+public class Trips {
```

Comment:
This is a single Trip, right? So call it Trip instead of Trips

Reply:
Also add an id for the Trip

Reply:
done